docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -41,6 +41,14 @@ You can configure the headers with:
 
 For more advanced header configuration, go to the [rendering section](/components/data-grid/rendering/#header-cell).
 
+## Column menu
```

Comment:
What do you think about documenting it just after `## Custom column types`? It doesn't seem that this section is important enough to be high in the document.

In the future, we will likely document how to customize the menu, e.g. add items.

Reply:
Sure, I put it there because of the alphabetical order, I'll move it ;)